backend/cloud_inquisitor/__init__.py

```diff
@@ -228,14 +26,19 @@ def get_aws_session(account):
     Returns:
         :obj:`boto3:boto3.session.Session`
     """
+    from cloud_inquisitor.config import dbconfig
+
     # If no keys are on supplied for the account, use sts.assume_role instead
-    if not api_access_key or not api_secret_key:
+    if not all((app_config.aws_api.access_key, app_config.aws_api.secret_key)):
```

Comment:
For ifs like this, I prefer having the common case be the if, and the uncommon case the else part of my if statements. In this case the most common is not having those specified, since we use instance profiles